src/pages/index.js

```diff
@@ -107,32 +134,83 @@ function closeModalByEscape(event) {
 // Handle Edit Form Submit
 function handleEditFormSubmit(evt) {
   evt.preventDefault();
-  profileName.textContent = profileNameInput.value;
-  profileDescription.textContent = profileDescriptionInput.value;
-  closeModal(editModal);
+  const saveButton = editForm.querySelector(".modal__submit-btn");
+  saveButton.textContent = "Saving...";
+
+  api
+    .editUserInfo({
+      name: profileNameInput.value,
+      about: profileDescriptionInput.value,
+    })
+    .then((data) => {
+      profileName.textContent = data.name;
+      profileDescription.textContent = data.about;
+      saveButton.textContent = "Save";
+      closeModal(editModal);
+    })
+    .catch((err) => {
+      saveButton.textContent = "Save";
```

Comment:
`saveButton.textContent = "Save";`

You need to **only** return the **default** button text in the `finally` block, so that the button could get the text regardless of whether the request succeeds or fails. You don't need to duplicate it in `then` and `catch`